### DISCUSSION TITLE
skip checkup error running Xcode 15.5

### DISCUSSION THREAD

**Discussion by fjeschke01** [skiptools/skip#176] [General]
![Screenshot 2024-07-21 at 6 12 07 PM](https://github.com/user-attachments/assets/e8f7d5e6-2d4c-4624-8a50-84afd590c10e)
All my software updates show complete
![Screenshot 2024-07-21 at 6 12 52 PM](https://github.com/user-attachments/assets/6accfebf-a463-46a8-9f18-52bacb68b492)
But it's giving me a problem with the Xcode version

**Comment by marcprux** [comment]
> Xcode version: error executing xcodebuild (install from: https://developer.apple.com/xcode/)

Is Xcode installed?

**Reply by fjeschke01** [reply]
Yes Sir

**Reply by fjeschke01** [reply]
![Screenshot 2024-07-21 at 6 41 44 PM](https://github.com/user-attachments/assets/9f02e1b6-41ba-4423-b705-481ec8f1aa05)

**Comment by marcprux** [comment]
Odd … it seems like it isn't found. Can you post the logfile that the `skip checkup` command references (/tmp/skip-checkup-2024-07-22T01:09:00Z.txt)?

**Reply by fjeschke01** [reply]
[skip-checkup-2024-07-22T01:09:00Z.txt](https://github.com/user-attachments/files/16327495/skip-checkup-2024-07-22T01.09.00Z.txt)

**Reply by fjeschke01** [reply]
After doing a searching the skip-checkup file for "Xcode" it seems to me that I need to remove Xcode and all related files and reinstall.

**Comment by marcprux** [comment]
It does seem like something is amiss with your Xcode install:

> warning: could not determine XCTest paths: terminated(1): /usr/bin/xcrun --sdk macosx --show-sdk-platform-path output:
>     xcrun: error: unable to lookup item 'PlatformPath' from command line tools installation
>     xcrun: error: unable to lookup item 'PlatformPath' in SDK '/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk'

Can you try running `xcode-select --install` and see if that fixes the issue?

**Reply by fjeschke01** [reply]
![Screenshot 2024-07-22 at 7 51 30 AM](https://github.com/user-attachments/assets/b643f6c4-1d33-4c0e-b894-18cfe5a6fe26)

**Reply by T1esto** [reply]
Once Xcode is installed, you need to set it as the active developer directory using the 
sudo xcode-select --switch /Applications/Xcode.app/Contents/Developer

if Xcode.app is named something else e.g "Xcode-15.4.app" 
sudo xcode-select --switch /Applications/Xcode-15.4.app/Contents/Developer
